env LC_ALL=C TZ=UTC0 diff -Naur gdal-1.7.1/data/gcs.csv gdal-1.7.1.patch/data/gcs.csv
--- gdal-1.7.1/data/gcs.csv	2010-03-30 17:45:17.000000000 +0000
+++ gdal-1.7.1.patch/data/gcs.csv	2010-03-30 17:48:26.000000000 +0000
@@ -204,7 +204,7 @@
 4274,Datum 73,6274,Datum 73,6274,9122,7022,8901,1,0,6422,,,,,,,,
 4275,NTF,6275,Nouvelle Triangulation Francaise,6275,9122,7011,8901,1,0,6422,9603,-168,-60,320,,,,
 4276,NSWC 9Z-2,6276,NSWC 9Z-2,6276,9122,7025,8901,1,0,6422,,,,,,,,
-4277,OSGB 1936,6277,OSGB 1936,6277,9122,7001,8901,1,0,6422,,,,,,,,
+4277,OSGB 1936,6277,OSGB 1936,6277,9122,7001,8901,1,0,9606,446.448,-125.157,542.060,0.1502,0.2470,0.8421,-20.4894
 4278,OSGB70,6278,"OSGB 1970 (SN)",6278,9122,7001,8901,1,0,6422,,,,,,,,
 4279,"OS(SN)80",6279,"OS (SN) 1980",6279,9122,7001,8901,1,0,6422,,,,,,,,
 4280,Padang,6280,Padang 1884,6280,9122,7004,8901,1,0,6422,,,,,,,,
@@ -225,7 +225,7 @@
 4296,Sudan,6296,Sudan,6296,9108,7011,8901,1,1,6402,,,,,,,,
 4297,Tananarive,6297,Tananarive 1925,6297,9122,7022,8901,1,0,6422,9603,-189,-242,-91,,,,
 4298,Timbalai 1948,6298,Timbalai 1948,6298,9122,7016,8901,1,0,6422,,,,,,,,
-4299,TM65,6299,TM65,6299,9122,7002,8901,1,0,6422,,,,,,,,
+4299,TM65,6299,TM65,6299,9122,7002,8901,1,0,9606,482.530,-130.596,564.557,-1.042,-0.214,-0.631,8.15
 4300,TM75,6300,Geodetic Datum of 1965,6300,9122,7002,8901,1,0,6422,,,,,,,,
 4301,Tokyo,6301,Tokyo,6301,9122,7004,8901,1,0,6422,,,,,,,,
 4302,Trinidad 1903,6302,Trinidad 1903,6302,9122,7007,8901,1,0,6422,,,,,,,,
